lib/Rarbg/torrentapi.pm

```diff
@@ -230,7 +239,79 @@ This is the Imdb id (http://imdb.com) in the form 'tt123456'
 
 Category can be quite confusing.
 It accepts 'tv' and 'movies'. But, for the rest of categories only accepts its id numbers (or a semi-colon separated list of them).
-Check Rarbg website to see what those are. They are not documented anywhere.
+
+=over 4
+
+=item * XXX (18+)
```

Comment:
They render up as bullet points, so that's ok.
put the number on the same line as you did for the README file.

Something like this: `=item * Movies/x264 => 17`